main.cpp

```diff
@@ -31,6 +32,7 @@ using VectorType = std::valarray<double>;
 double log_likelihood (const SampleType &prm)
 {
   // S3 data
+  /*
```

Comment:
If you don't need something any more, just remove it. You can always get it back through the git history.